examples_utils/notebook_logging/__init__.py

```diff
@@ -1,3 +1,12 @@
 # Copyright (c) 2023 Graphcore Ltd. All rights reserved.
 
+import subprocess
+
+from pathlib import Path
 from .gc_logger import *
+
+# Help IPython find our custom extension
+extension_path = Path(__file__).parent.joinpath("gc_logger.py").resolve()
+destination_path = Path("/root/.ipython/extensions").resolve()
+
+subprocess.run(f"cp {extension_path} {destination_path}", shell=True)
```

Comment:
Could this happen on installation? So that we can use the extension directly?

Reply:
I didnt look into that, I didnt think that would be possible...

I'll take a look now

Reply:
Tested out running custom build steps in setup.py, dosent seem to work... might test in the future but not now